volttrontesting/platform/test_platform_rmq.py

```diff
@@ -328,26 +296,21 @@ def test_expired_ca_cert_after_vstart(request, instance):
                 vip_identity="listener2", start=True)
             pytest.fail("Agent install should fail")
         except Exception as e:
-            print("Exception:", e)
-            assert True
-
-        # Can't find status. Essentially we have to create CA and reissue all client certs.
-        try:
-            instance.is_agent_running(agent)
-        except Exception as e:
+            # rabbitmq log will show a certificate expired error
```

Comment:
Why aren't we using the pytest with exception wrapper rather than this?

Reply:
I catch exception explicitly so that I can check the exception message